### PR TITLE
Paul/state picker

### PR DESCRIPTION
**Pull request by paullinator**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [x] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206540710532758

### CODE REVIEW

File: src/locales/en_US.ts [hunk 1]
```diff
@@ -865,6 +865,7 @@ const strings = {
   choose_your_wallet: 'Choose Your Wallet',
   error_creating_wallets:
     'Network timeout occurred trying to create and backup wallets. Please check your network connection and restart app to retry wallet creation.',
+  error_must_select_state_province: 'Must select state/province for %s',
```
Comment (Jon-edge) [inline src/locales/en_US.ts:868]: Needs to follow explicit _1s %1$s convention

**Review by Jon-edge — CHANGES_REQUESTED** [review]
Minor changes

**Review by samholmes — CHANGES_REQUESTED** [review]
One more change request about the use of `useHandler`. Otherwise, looks good to me.

**Pull request merged**